metabase/metabase
# Adds paid plan-specific links

**Adds**:
1. A link to `Contact support` when running on a paid plan
2. A link to the store when **not** running on a paid plan

**How to test**:

Visit `/admin`. The store link:
- should be visible when running OSS
- should be hidden when running EE
- should be hidden when running Cloud

Visit `/admin/troubleshooting/help`. A link to contact support:
- should be hidden when running OSS
- should be visible when running EE
- should be visible when running Cloud

To run enterprise:
```
MB_EDITION=ee yarn dev
```

To run cloud either:
- Modify `isHosted` implementation to pass locally
- Rely on cypress tests

<img width="195" alt="Screen Shot 2021-09-03 at 4 15 05 pm" src="https://user-images.githubusercontent.com/8542534/132011230-0c2ba6bd-9699-468c-b192-48b47c3b2df5.png">
<img width="453" alt="Screen Shot 2021-09-03 at 4 16 20 pm" src="https://user-images.githubusercontent.com/8542534/132011234-75dfc0b3-75cd-4e5d-9f1e-57d6841372b4.png">
